src/Atlas2DSlice.ts

```diff
@@ -178,9 +178,10 @@ export default class Atlas2DSlice implements VolumeRenderImpl {
 
       const sliceInBounds = this.updateSlice();
       if (sliceInBounds) {
-        const sliceRatio = Math.floor(this.settings.zSlice) / this.volume.imageInfo.volumeSize.z;
+        const sliceLowerBound = Math.floor(this.settings.zSlice) / this.volume.imageInfo.volumeSize.z;
+        const sliceUpperBound = (Math.floor(this.settings.zSlice) + 1) / this.volume.imageInfo.volumeSize.z;
         this.volume.updateRequiredData({
-          subregion: new Box3(new Vector3(0, 0, sliceRatio), new Vector3(1, 1, sliceRatio)),
+          subregion: new Box3(new Vector3(0, 0, sliceLowerBound), new Vector3(1, 1, sliceUpperBound)),
```

Comment:
interesting, I thought we had special handling for when you used the same z value

Reply:
There is within the loader, but this change addresses a bug in how scale levels are picked. If min and max Z are the same when picking scale levels, we ended up calculating the required tile size from X and Y, then searching for a scale level that can fit 0 of those tiles. Perhaps the better way to handle this would be enforcing a maximum of 1 when doing that calculation?

Reply:
New changes do the min-checking I describe above, though I'm on the fence about whether to revert this change.

Reply:
I figure both are good

Reply:
The one odd thing here is that it's hard to tell whether the caller intends zSlice or zSlice+1, just by looking at the bounds. You almost want to do zSlice-0.5, zSlice+0.5 .  I don't know if that bias is commented anywhere.